src/app/components/form-fields/dropdown/dropdown.component.html

```diff
@@ -1,5 +1,7 @@
-<label>{{label}}</label>
+<label for="{{label}}">{{label}}</label>
   <select
+    name="{{label}}"
+    id="{{label}}"
```

Comment:
Let's assume the label is `OH MY GOD EDWIN !` .... is it still right ?

Reply:
This a lazy approach that won't work here. You better generate a true ID here. A quick searhc on stack overflow can save you a lot of time https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid
Good think is that, the id will be uniq regardless the amount of time your component will be used

Reply:
So I should use a different "@Input() name: string" property that's populated from the parent component ?

Reply:
`@input() name: string` is very confusing ...as is i feel like you want to feed a input name property like. Here we are just talking about `id`, that's it